# HG changeset patch
# User Rik <rik@octave.org>
# Date 1437666853 25200
#      Thu Jul 23 08:54:13 2015 -0700
# Node ID f248847b507126a6b180189bd60258b559fb7e72
# Parent  e2feb806332a65d6bbefa2f59d9946b7904abe0e
plotimages.m: Remove workaround for polar plot rticks.

* plotimages.m: Don't manually set rtick values now that they are correctly
calculated by polar().

diff --git a/doc/interpreter/plotimages.m b/doc/interpreter/plotimages.m
--- a/doc/interpreter/plotimages.m
+++ b/doc/interpreter/plotimages.m
@@ -55,17 +55,16 @@ function plotimages (d, nm, typ)
     errorbar (x, sin (x), yl, yu);
     axis ([0, 10, -1.1, 1.1]);
     xlabel ("x");
     ylabel ("sin (x)");
     title ("Errorbar plot of sin (x)");
     print (outfile, d_typ);
   elseif (strcmp (nm, "polar"))
     polar (0:0.1:10*pi, 0:0.1:10*pi);
-    set (gca, "rtick", [0:10:40]);
     title ("Example polar plot from 0 to 10*pi");
     print (outfile, d_typ);
   elseif (strcmp (nm, "mesh"))
     tx = ty = linspace (-8, 8, 41)';
     [xx, yy] = meshgrid (tx, ty);
     r = sqrt (xx .^ 2 + yy .^ 2) + eps;
     tz = sin (r) ./ r;
     mesh (tx, ty, tz);
